btor/src/boolector_term.cpp

```diff
@@ -267,9 +265,27 @@ Sort BoolectorTerm::get_sort() const
   return sort;
 }
 
+bool BoolectorTerm::is_symbol() const
+{
+  // functions, parameters, and symbolic constants are all symbols
+  auto bkind = bn->kind;
+  return !negated
```

Comment:
a side note: we should have different naming convention for data members.

Reply:
i am referring to `negated` here

Reply:
Ahh like `negated_`? Yeah that's not used anywhere in the codebase, but would probably be a good idea.